pkg/asset/installconfig/aws/subnet.go

```diff
@@ -164,7 +164,8 @@ func subnets(ctx context.Context, session *session.Session, region string, ids [
 		}
 
 		// AWS Local Zones are grouped as Edge subnets
-		if meta.Zone.Type == typesaws.LocalZoneType {
+		if meta.Zone.Type == typesaws.LocalZoneType ||
+			meta.Zone.Type == typesaws.WavelengtyZoneType {
```

Comment:
ooh, I just noticed it looks like we merged (in a previous commit) a typo here?

```suggestion
			meta.Zone.Type == typesaws.WavelengthZoneType {
```
we could fix here or in another pr, either is fine